doc/source/security/overview-master-key.rst

```diff
@@ -0,0 +1,380 @@
+   :default: OFF
+
+This variable simultaneously turns on the encryption of files used by InnoDB for
+full text search using parallel sorting, building indexes using merge sort, and
+online DDL logs created by InnoDB for online DDL.
+
+.. _data-at-rest-encryption.variable.innodb-scrub-log:
+
+.. variable:: innodb_scrub_log
+
+   :version 5.7.23-24: Implemented
+   :cli: ``--innodb-scrub-log``
+   :dyn: Yes
+   :scope: Global
+   :vartype: Boolean
+   :default: ``OFF``
+
+Specifies if data scrubbing should be automatically applied to the redo log.
+
+
+.. variable:: innodb_scrub_log_speed
+
+   :version 5.7.23-24: Implemented
+   :cli: ``--innodb-scrub-log-speed``
+   :dyn: Yes
+   :scope: Global
+   :vartype: Text
+   :default:
+
+Specifies the velocity of data scrubbing (writing dummy redo log records) in bytes per second.
+
+
+For `innodb_redo_log_encrypt`, the "ON" value is an alias for ``MASTER_KEY``
+value.
+
+The redo log data is encrypted when it is written to disk and decrypted when the
+data is read from disk. Any redo log data in memory is unencrypted.
+
+When `innodb_redo_log_encrypt` is enabled, any existing redo log pages stay
+unencrypted, and new pages are encrypted.
+
+If the redo log encryption is enabled, you must load the appropriate keyring
+plugin and encryption key to perform a normal restart.
+
+When `innodb_redo_log_encrypt` is
+disabled, any existing pages remain encrypted, and new pages are not encrypted.
+
+An attempt to encrypt the redo log fails if one of the following conditions is
+true:
+
+* Server started with no keyring specified
+
+* A different redo log encryption method is defined then what was previously
+  used on the same server.
+
+.. _data-at-rest-encryption.undo-tablespace:
+.. _undo-log-encryption:
+
+Undo Log Encryption
+--------------------
+
+:Availability: This feature is **Experimental** quality.
+
+Undo log data is encrypted using the
+:variable:`innodb_undo_log_encrypt` option. You can edit this variable
+in the configuration file, as a startup parameter, or during runtime as a global
+variable.
+
+.. variable:: innodb_undo_log_encrypt
+
+    :version 5.7.23-24: Implemented
+    :cli: ``--innodb_undo-log_encrypt``
+    :dyn: Yes
+    :scope: Global
+    :vartype: Boolean
+    :default: OFF
+
+Defines if an undo log data is encrypted. The undo log data encryption is disabled by default.
+
+Undo log data is encrypted when the data is read to disk. The data is decrypted
+when the data is read from disk. When the undo log data is in memory, the data
+is unencrypted.
+
+The undo log data is encrypted and decrypted with the tablespace encryption key,
+which is stored in the undo log file header.
+
+.. note::
+
+    If you disable encryption, any encrypted undo data remains encrypted. To
+    remove this data, truncate the undo tablespace.
+
+.. _binlog-encryption:
+
+Binlog Encryption
+------------------
+
+Enable the ``binlog_encryption`` option to encrypt new binary log files and
```

Comment:
As I have commented earlier this is enabled by setting encrypt_binlog. Later in this paragraph, you describe correctly encrypt_binlog variable.